terraform/modules/bucket/variables.tf

```diff
@@ -5,6 +5,16 @@ variable "name" {
 
 variable "cross_account_read_roles" {
   description = "Roles in other accounts that need read access to this S3 bucket"
-  type        = list
+  type        = list(any)
   default     = []
 }
+
+variable "app" {
+  description = "The name of the application"
+  type        = string
+}
+
+variable "env" {
+  description = "The environment name"
+  type        = string
+}
```

Comment:
Please add validation as in https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/6cc555cad64f7cb69d2a417f13d34657316b9c1a/terraform/modules/vpc/variables.tf